students/Kollii/lesson06/test_mailroom4.py

```diff
@@ -0,0 +1,48 @@
+import mailroom4 as mailroom
+import os.path
+import pathlib
+
+
+mailroom.donor_dict = mailroom.donors()
+
+def test_donors_history():
+    """ Test History of dictionary of donors created """
+    history = mailroom.donors()
+    assert history['Jeff Bezos'] == [877.33]
+    assert history['William Gates, III'] == [653772.32, 12.17]
+
+def test_donors_list():
+    """ Test all donor names are returned"""
+    
+    thislist = ('William Gates, III\n Jeff Bezos\n Paul Allen\n Mark Zuckerberg')
+    assert mailroom.donors_list() == thislist
```

Comment:
a common pattern is to use a variable like `expected` - it's clear here in these simple tests but in more complicated tests it can help clarify. for example:
```
`ssert mailroom.donors_list() == expected
```